src/oc/fileio.cpp

```diff
@@ -897,6 +898,13 @@ static int vnrnpy_pr_stdoe(FILE* stream, const char *fmt, va_list ap) {
 
     /* Determine required size */
     va_list apc;
+#ifndef va_copy
+#if defined(__GNUC__) || defined(__clang__)
+#define va_copy(dest, src) __builtin_va_copy(dest, src)
+#else
+#define va_copy(dest, src) (dest = src)
+#endif
+#endif
```

Comment:
to clarify - was this old implementation? (specifically wondering about assignment)

Reply:
This is new stuff, I put it there because older gcc from CI was failing.

Reply:
I am sure you have checked this but I was just curious about va_copy doing just (pointer) assignment. Not sure about it's usage but I hope nothing critical here.

Reply:
Those are not pointers, but `va_list`s